text/0002-searchable-display-attributes-behaviors.md

```diff
@@ -0,0 +1,123 @@
+- Title: Searchable Attributes and Displayed Attributes behaviors
+- Start Date: 2020-11-02
+- specification PR: meilisearch/specifications#3
+- Meilisearch Issue: 
+
+# `searchable_attributes` & `displayed_attributes` behaviors
+
+## First section: Feature Description and Interaction
+
+This first part has a general audience. It should be as little technical as possible (think user-level). This section contains 4 sub-sections:
+
+### Summary
+
+Ever since the beginning, the design of `searchableAttributes` and `displayedAttributes` as shown flaws. The introduction of the wildcard tried to adress some of those flaws, but there are still issues that this specs tries to correct.
+
+Searchable attributes have several uses:
+- know in which fields we can search.
+- know the importance of fields for the criterion `Attributes`.
+
+The main issues we want to adress here are:
+- The question of `searchableAttributes` order when the wildcard is used.
+- The fact that already known attributes (e.g from facets), are not indexed correctly is put before the documents (bug), need to be taken into account in the design.
+
+Is it relevant to mix this 2 information when we have wildcards?
+
+### Explanation
+
+#### default values
+
+`searchable_attributes`: *   
+`displayed_attributes`: *
+
+#### wildcard (*) & `searchable_attributes` order
+
+> When the searchable attributes value is set to ["*"] the priority of the documents attributes is undefined.
+
+By default searchable attributes should be in the order of the first indexed document, for example:
+```json
+{
+  "title": "many the fish",
+  "description": "it is awfull to be liked by many"
+}
+```
+default `searchable_attributes` order should be `["title", "description"]` where `"title" > "description"`
+
+When a wildcard is set we should have the default behavior.
+
+#### `searchable_attributes` & `displayed_attributes`
+
+what is the behavior for a field that:
+- > **is in `searchable_attributes` and in `displayed_attributes`:**
+  > meilisearch will use the field for a search and the field will be returned in search results
+- > **is in `searchable_attributes` but not in `displayed_attributes`:**
+  > meilisearch will use the field for a search but the field will **not** be returned in search results
+- > **is not in `searchable_attributes` but in `displayed_attributes`:**
+  > meilisearch will **not** use the field for a search but the field will be returned in search results
+- > **is not in `searchable_attributes` and not in `displayed_attributes`:**
+  > meilisearch will **not** use the field for a search and the field will **not** be returned in search results. The field is unused but should be kept in case of it is added in `searchable_attributes` or `displayed_attributes`.
```

Comment:
```suggestion
  > meilisearch will **not** use the field for a search and the field will **not** be returned in search results. The field is unused but should be kept in case of is added in `searchable_attributes` or `displayed_attributes`, or present in a new document.
```